source/localizable/tutorial/installing-addons.md

```diff
@@ -0,0 +1,85 @@
+Ember has a rich ecosystem of addons that can be easily added to projects.
+Addons provide a wide range of functionality to projects, often saving time and letting you focus on your project's value.
+
+To browse addons, visit the [Ember Observer](https://emberobserver.com/) website.  It catalogs and categorizes ember addons that have been published to NPM and assigns them a score based on a variety of criteria.
+
+For Super Rentals, we'll take advantage of two addons: [ember-cli-tutorial-style](https://github.com/toddjordan/ember-cli-tutorial-style) and [ember-cli-mirage](http://www.ember-cli-mirage.com/).
+
+### ember-cli-tutorial-style
+
+Instead of having you copy/paste in CSS to style Super Rentals, we've created an addon called [ember-cli-tutorial-style](https://github.com/toddjordan/ember-cli-tutorial-style) that instantly adds CSS to the tutorial.
+The addon works by creating a file called `ember-tutorial.css` and putting that file in the application's `/vendor` directory.
+At compile time the contents of the css file is automatically concatenated to the `vendor.css` file which is imported into the application's `apps/index.html`.
+By creating the file in the `vendor` directory, the addon is exposing it to the developer for potential customization (in case we want to tweak the styles).
```

Comment:
I think we can skip some of this explanatory text. Maybe something along these lines instead?

As Ember-CLI runs, it takes the addon's CSS file and puts it in `vendor.css` (which is referenced in `/app/index.html`). We can make additional style tweaks in `/app/styles/app.css`